specs/specs.go

```diff
@@ -98,14 +101,32 @@ type StringAssert struct {
 	NotContains *string `json:"notContains,omitempty"`
 }
 
-func LoadSuite(path string) *Suite {
-	data, err := ioutil.ReadFile(path)
+type TemplateSpec struct {
+	Vars map[string]string
+	Env  map[string]string
+}
+
+func getEnvs() map[string]string {
+	envs := make(map[string]string)
+	for _, element := range os.Environ() {
+		splits := strings.Split(element, "=")
+		envs[splits[0]] = splits[1]
+	}
+	return envs
+}
+
+func LoadSuite(path string, vars *map[string]string) *Suite {
+	templateFile, err := ioutil.ReadFile(path)
 	check(err)
 
+	env := getEnvs()
```

Comment:
for testability and encapsulation, this should be passed as a parameter to this function as well